packages/common/src/types/field.ts

```diff
@@ -445,6 +445,10 @@ export interface Metric extends Field {
     formatOptions?: CustomFormat;
     dimensionReference?: string; // field id of the dimension this metric is based on
     requiredAttributes?: Record<string, string | string[]>; // Required attributes for the dimension this metric is based on
+    defaultTimeDimension?: {
+        field: string;
+        interval: TimeFrames;
+    };
```

Comment:
do we need to have this on a field level?

Reply:
https://github.com/lightdash/lightdash/pull/12505/commits/7381dc800cb0c657747504f0381dba988aebbbde